### ISSUE TITLE
[BUG] Android Pie on Mi A2 Lite

### ISSUE THREAD

**Issue by zkeu** [ElderDrivers/EdXposed#224]
什么情况/What happened?

device: Mi A2 Lite, stock rom Android PIE

Xposed模块列表/Xposed Module List

Gravity Box [P]

Magisk模块列表/Magisk Module List

NotchKiller, Pix3lify (FULL), Riru - Core. Riru - Ed Xposed, Youtube Vanced

EdXposed及Riru版本/Versions of EdXposed and Riru

EdXposed: v0.3.1.7 beta

Riru: v18

If EdXposed v0.3.1.7_beta magisk module is activated, mobile data does not work.

It does not matter which Riru core version I use. If I use EdXposed v0.2.9.9_beta2, then mobila data works, but this version of EdXposed is really buggy. 

I am use stock Pie rom V10.0.7 for Mi A2 Lite, but it was the same with V10.0.4.

**Comment by zkeu** [comment]
I think I have found the problem. I seems that EdXposed module (I used v0.3.1.7 and also the v0.4.0.1)  and Camera 2 API are somehow interfering with each other.

It does not matter if I use Magisk module Camera2_API_Enabler-v1.0.0 or activate camera 2 API threw Su ADB with:
1. setprop persist.camera.HAL3.enabled 1
2. setprop persist.camera.eis.enabled 1

The result is always the same. My mobile data won't work. If I disable Camera 2 API, then mobile data works.
